app/services/notifications/handler.rb

```diff
@@ -0,0 +1,32 @@
+module Notifications
+  class Handler
+    attr_reader :activity
+
+    def initialize(activity:)
+      @activity = activity
+    end
+
+    def call
+      broadcast_messages
```

Comment:
What happens with this fails?  We don't want to add a record to the inbox in that case.